shared/gh/js/views/gh.admin-batch-edit-date.js

```diff
@@ -169,7 +169,7 @@ define(['lodash', 'moment', 'gh.core', 'gh.api.config'], function(_, moment, gh,
                         'displayName': $('.gh-jeditable-series-title').text(),
                         'end': gh.utils.convertUnixDatetoISODate(moment(endDate).toISOString()),
                         'location': '',
-                        'notes': 'Lecture',
+                        'type': 'Lecture',
```

Comment:
Maybe we can use the default value from the configuration here: `gh.config.events.default`